components/builder-web/app/origin/origin-page/origin-integrations-tab/_origin-integrations-tab.component.scss

```diff
@@ -16,42 +16,107 @@
     margin: 0 auto rem(30);
   }
 
-  ul {
-    margin-top: 18px;
-    border-top: 1px solid $very-light-gray;
+  .registries {
+    .mat-grid-tile {
+      overflow: initial;
 
-    li {
-      @include row;
-      border-bottom: 1px solid $very-light-gray;
-      padding: 12px;
-      font-size: rem(12);
+      button {
+        width: 355px;
+        height: 54px;
+        position: relative;
 
-      hab-icon {
-        width: 16px;
-        height: 16px;
-      }
+        &#docker,
+        &#amazon {
+          box-shadow: 0 2px 15px 0 rgba(0,0,0,0.08);
+        }
 
-      .name {
-        @include span-columns(10);
-        font-weight: 600;
+        span:last-child {
+          line-height: 55px;
+          float: left;
+        }
 
         hab-icon {
-          margin-right: 6px;
+          width: 24px;
```

Comment:
These heights and widths should always match; they can be anything, but their numbers should always be the same unless you have some need to change how they interact with the viewport.